sklearn/ensemble/_voting.py

```diff
@@ -637,6 +637,12 @@ class VotingRegressor(RegressorMixin, _BaseVoting):
     >>> er = er.fit(X, y)
     >>> len(er.estimators_)
     2
+
+    For a detailed example of utilizing
```

Comment:
It would be better to be more succinct such as: 

For a more detailed example, refer to :ref:`sphx_glr_auto_examples_ensemble_plot_voting_regressor.py`.